framework/test/jest/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-	"extends": ["plugin:jest/recommended"],
+	"extends": ["lisk-base/jest"],
```

Comment:
you can remove plugins, env and rules properties since now they are not affected by mocha rules.

Reply:
I'm not sure if plugins would cause a problem but shouldn't :D let's see. But rest is definitely fine.